docs/PLUGINS.md

```diff
@@ -233,35 +233,37 @@ to load.
 #### Configuration
 
 Plugins are configured using the `[plugins]` section of containerd's config.
-Every plugin can have its own section using the pattern `[plugins.<plugin id>]`.
+Every plugin can have its own section using the pattern `[plugins."<plugin type>.<plugin id>"]`.
 
 example configuration
 ```
+version = 2
+
 [plugins]
-  [plugins.cgroups]
+  [plugins."io.containerd.monitor.v1.cgroups"]
     no_prometheus = false
-  [plugins.cri]
+  [plugins."io.containerd.grpc.v1.cri"]
     stream_server_address = ""
     stream_server_port = "10010"
     enable_selinux = false
     sandbox_image = "k8s.gcr.io/pause:3.6"
     stats_collect_period = 10
     systemd_cgroup = false
-    [plugins.cri.containerd]
+    [plugins."io.containerd.grpc.v1.cri".containerd]
       snapshotter = "overlayfs"
-      [plugins.cri.containerd.default_runtime]
+      [plugins."io.containerd.grpc.v1.cri".containerd.default_runtime]
         runtime_type = "io.containerd.runtime.v1.linux"
         runtime_engine = ""
         runtime_root = ""
-      [plugins.cri.containerd.untrusted_workload_runtime]
+      [plugins."io.containerd.grpc.v1.cri".containerd.untrusted_workload_runtime]
         runtime_type = ""
         runtime_engine = ""
         runtime_root = ""
-    [plugins.cri.cni]
+    [plugins."io.containerd.grpc.v1.cri".cni]
       bin_dir = "/opt/cni/bin"
       conf_dir = "/etc/cni/net.d"
-    [plugins.cri.registry]
-      [plugins.cri.registry.mirrors]
-        [plugins.cri.registry.mirrors."docker.io"]
+    [plugins."io.containerd.grpc.v1.cri".registry]
```

Comment:
this is definitely worth updating, but I wonder if we should just remove the registry section as this is using the now-deprecated pattern for registry configuration? Or simply remove the mirrors/endpoint content and replace it with the config_path setting which is the new configuration method for registry hosts/mirrors.

Reply:
the example here is an old one.. and partial..  and there is no explanation about the contents... 

Suggest removing the cri parts... and leave cgroups as the example

Maybe mention how to run `containerd config default` / dump to see the current default / combined config

Reply:
@estesp @mikebrow Thank you!

> Suggest removing the cri parts... and leave cgroups as the example
> Maybe mention how to run containerd config default / dump to see the current default / combined config

I think your suggestion is good.
I'll remove `cri` part and use this snippet as a short example.
And mention about how to run `containerd config default / dump` to see the current default / combined config to tell how to get a full example.

Reply:
@estesp @mikebrow I've fixed this. https://github.com/containerd/containerd/pull/6613/commits/03a5e648bcd7e65bb3654c06a896d17968b597f2